sql/migrations/0001_initial.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE blocks ( 
```

Comment:
Could you please fix the indentation in this file? The content of `CREATE TABLE blocks` is aligned by 1 tab, `ballots` by mixed 4 spaces/1 tab, `layers` by 4 spaces

Reply:
didn't notice that, will fix